src/carousel/card.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import './card.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap-icons/font/bootstrap-icons.css';
+
+export default class Card extends Component {
```

Comment:
why class based React Components? We have been doing functional components in this class